### ISSUE TITLE
[bitnami/kafka] JMX Exporter gets OOM killed on Cgroups v2 system

### ISSUE THREAD

**Issue by Agalin** [bitnami/charts#13071]
### Name and Version

bitnami/kafka 19.0.1

### What steps will reproduce the bug?

* Deploy chart with jmx exporter enabled and memory limit set.
* See its memory usage grow until dead.

### What is the expected behavior?

The exporter limits its memory to the provided value.

### What do you see instead?

The exporter fails to read the limit and wants to use the whole system memory.

Visible in logs:
```
OpenJDK 64-Bit Server VM warning: Unable to open cgroup memory limit file /sys/fs/cgroup/memory/memory.limit_in_bytes (No such file or directory)
```

### Additional information

The image uses Java 8 which does support cgroups as a backport from Java 10. But Java 10 only supports cgroups v1. V2 support was added in Java 15 and backported to 11.0.16 (see [this issue](https://bugs.openjdk.org/browse/JDK-8285289) for details).

With Java updated, the container should respect limits on both Cgroups V1 and V2. But it's possible to work around this issue even without updating the image - by passing `-XX:MaxRAM` explicitly to JVM at startup.
The problem with this approach? The chart doesn't provide an option to customize the exporter's command. And it would need to be kept in sync between args and limits manually.

This is probably the source issue of https://github.com/prometheus/jmx_exporter/issues/690 too.

**Comment by migruiz4** [comment]
Hi @Agalin,

Thank you for reporting this issue. I have submitted this PR #13114 adding support for additional java args for the JMX exporter, so the -XX:MaxRAM can be passed to the jmx container.

In addition, we will be updating the Java version of the JMX exporter container from Java 8 to Java 11.
I will let you know once the updated image is released.

**Comment by Agalin** [comment]
Awesome! Thanks for the fast reaction. 😄

**Comment by migruiz4** [comment]
Hi @Agalin,

We have released the image `bitnami/jmx-exporter:0.17.2-debian-11-r11` which uses Java 11.0.17.

Could you please give it a try and confirm if the issue is resolved?

**Comment by Agalin** [comment]
Needed to remove `UseCGroupMemoryLimitForHeap` from args (it's no longer available). There is also a warning about `MaxRAMFraction` being deprecated in Java 10.

Memory limit is properly detected under both CGroups v1 and V2. With limits set to 1Gi JVM configures itself to:
```
    Max. Heap Size (Estimated): 989.88M
```
